pkg/backup/backup.go

```diff
@@ -112,13 +112,14 @@ func (ctx *backupContext) getResourceIncludesExcludes(helper discovery.Helper, i
 	return collections.GenerateIncludesExcludes(
 		includes,
 		excludes,
-		func(item string) (string, error) {
+		func(item string) string {
 			gr, err := helper.ResolveGroupResource(item)
 			if err != nil {
-				return "", err
+				ctx.log("Unable to resolve resource %q: %v", item, err)
```

Comment:
Why did you change this from returning an error to logging the error and returning a valid value (i'm guessing "" is valid in some circumstances)

Reply:
so that includes_excludes.GenerateIncludesExcludes(...) doesn't have to take a logger (right now it's glogging this error). See related change to that function to check for empty-strings.